.ci/schedule-daily.groovy

```diff
@@ -52,14 +52,16 @@ pipeline {
             string(name: 'registry', value: 'docker.elastic.co'),
             string(name: 'tag_prefix', value: 'observability-ci'),
             string(name: 'secret', value: 'secret/apm-team/ci/docker-registry/prod'),
+            booleanParam(name: 'apm_integration_testing', value: true),
+            booleanParam(name: 'apm_server', value: true),
+            booleanParam(name: 'helm_kubectl', value: true),
             booleanParam(name: 'nodejs', value: true),
+            booleanParam(name: 'opbot', value: true),
+            booleanParam(name: 'oracle_instant_client', value: true),
             booleanParam(name: 'python', value: true),
             booleanParam(name: 'ruby', value: true),
+            booleanParam(name: 'rum', value: true),
```

Comment:
What is the difference with https://github.com/elastic/apm-pipeline-library/blob/master/.ci/buildDockerImages.groovy, which is scheduled daily?

Reply:
I don't see any trigger there, besides the pipeline got a default `false` parameters, so even if it gets triggered it won't do anything.

`buildDockerImages.groovy` is only the how while `schedule-daily.groovy` is the `when`